tests/plugins/list/backspace.html

```diff
@@ -719,3 +719,133 @@
 		</li>
 	</ol>
 </textarea>
+
+<textarea id="delete_neighboring_lists1">
+	<ol>
+		<li>a</li>
+		<li>b</li>
+		<li>c^</li>
```

Comment:
So backspace is being triggered at this cursor location? Why doesn't `c` just disappear? Or is it delete that's being triggered?